packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js

```diff
@@ -219,7 +220,6 @@ function RevisionsButtons( {
 								</p>
 							) : (
 								<Button
-									disabled={ areStylesEqual }
```

Comment:
Confirmed that this can and should be removed because `areStylesEqual` will never be true since it's already `false` based on the ternary logic this is nested in.